### ISSUE TITLE
We should score packages based on whether or not their changelog honors our spec.

### ISSUE THREAD

**Issue by sigurdm** [dart-lang/pana#1407]

**Comment by sigurdm** [comment]
Blocked on https://github.com/dart-lang/pana/issues/1406

**Comment by sigurdm** [comment]
Closing in favor of https://github.com/dart-lang/pana/issues/1406
